packages/chain-adapters/src/utxo/UTXOChainAdapter.ts

```diff
@@ -380,12 +385,12 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
 
   async getAddress({
     wallet,
-    bip44Params = ChainAdapter.defaultBIP44Params,
+    bip44Params,
```

Comment:
IDK if `this` works here; if not the parameter should be `undefined` or something and then the value subbed in below
```suggestion
    bip44Params = this.defaultBIP44Params,
```